src/main/java/io/kuzzle/sdk/API/Controllers/AuthController.java

```diff
@@ -208,7 +205,7 @@ public AuthController(final Kuzzle kuzzle) {
     query
         .put("controller", "auth")
         .put("action", "refreshToken")
-        .put("expiresIn", expiresIn);
+        .put("expiresIn", (expiresIn == null ? "1h" : expiresIn));
```

Comment:
`expiresIn` is optional and a default value can be set in the `kuzzlerc`, the SDK shouldn't have his own default value
```suggestion
        .put("expiresIn", expiresIn);
```

Reply:
Since when? According to the API documentation there is an expiresIn (https://docs.kuzzle.io/core/2/api/controllers/auth/login/) and our most up to date SDK handle the expiresIn https://docs.kuzzle.io/sdk/js/7/controllers/auth/login/ , https://github.com/kuzzleio/sdk-javascript/blob/master/src/controllers/auth.js#L180